modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -430,62 +430,35 @@ def date_of_release
 
       def disability_attributes
         @pdf_data[:data][:attributes][:claimInformation] = {}
-        @pdf_data[:data][:attributes][:claimInformation].merge!(
-          { disabilities: [] }
-        )
         conditions_related_to_exposure?
-
-        disabilities = transform_disabilities
-
-        details = disabilities[:data][:attributes][:claimInformation][:disabilities].map(
-          &:deep_symbolize_keys
-        )
-        @pdf_data[:data][:attributes][:claimInformation][:disabilities] = details
-
+        @pdf_data[:data][:attributes][:claimInformation][:disabilities] = transform_disabilities
         @pdf_data[:data][:attributes].delete(:disabilities)
```

Comment:
a lot of this was irrelevant. I wanted to remove the `.delete` ideally (as well) but we embed the entire object in to attributes early so it was a much heavier lift.